.github/workflows/checkForTranslations.yaml

```diff
@@ -0,0 +1,31 @@
+# https://crontab.guru/crontab.5.html
```

Comment:
Placing this file in `.github/workflows` has an unfortunate side effect that this action would be run for addonTemplate repo which is not what we want. Perhaps these sample actions should be moved to something like `GitHubActions/sampleActions` and documented in the readme.

Reply:
Disabling this action for add-on template repository would work but only for this repo. If someone uses this template as is and their add-on is not registered in the translations system they would be getting failures by default. I still maintain that moving these files from `.github/workflows` and documenting each of these actions in the readme is the simplest way to deal with the problem though I don't have any authority over this repository obviously.